src/main/java/org/apache/maven/plugins/dependency/tree/TreeMojo.java

```diff
@@ -380,6 +380,8 @@ public DependencyNodeVisitor getSerializingDependencyNodeVisitor(Writer writer)
             return new TGFDependencyNodeVisitor(writer);
         } else if ("dot".equals(outputType)) {
             return new DOTDependencyNodeVisitor(writer);
+        } else if ("json".equals(outputType)) {
+            return new JsonDependencyNodeVisitor(writer);
         } else {
             return new SerializingDependencyNodeVisitor(writer, toGraphTokens(tokens));
```

Comment:
In the other PR someone asked for a switch case instead. Shall I convert this to a switch statement?

Reply:
no, this is fine